src/main/java/org/opentripplanner/util/model/EncodedPolylineBean.java

```diff
@@ -9,80 +9,18 @@
  * polyline algorithm format</a>
  */
 
-public class EncodedPolylineBean implements Serializable {
-
-  private static final long serialVersionUID = 1L;
-
-  private String points;
-
-  private String levels;
-
-  private int length;
-
-  public EncodedPolylineBean() {}
-
-  public EncodedPolylineBean(String points, String levels, int length) {
-    this.points = points;
-    this.levels = levels;
-    this.length = length;
-  }
-
+public record EncodedPolylineBean(String points, int length) implements Serializable {
```

Comment:
Do we want to keep using the suffix `Bean`? I know this will make the diff much larger but I think it can go.